README.rst

```diff
@@ -29,6 +29,7 @@ Features
 * Stripe Sources
 * Stripe v2 and v3 support
 * Tested with Stripe API `2019-05-16` (see https://dj-stripe.readthedocs.io/en/latest/api_versions.html )
+* Supports `SCA regulations, Checkout Sessions, and Payment Intents <https://github.com/dj-stripe/dj-stripe/commit/23ab8577e314d9a4720ce401ab743f3a4f382b01>`_
```

Comment:
Go ahead and move this directly after `Stripe v2 and v3 support`.

Also remove the link to the commit; people don't necessarily need to see the code. We'll make sure the release notes are linked properly.